### DISCUSSION TITLE
Not receiving my Academic Benifits after 3 days of approval

### DISCUSSION THREAD

**Discussion by tasnim-rizvy** [community/community#145827] [GitHub Education]
### Hi everyone,

I got approved for my student account on November 26, 2024, but I still haven't received the Student Developer Pack. How long will I have to wait?

![Screenshot from 2024-11-29 09-00-06](https://github.com/user-attachments/assets/682e42f4-793b-4969-b6c1-69ee3cfa1420)

![Screenshot from 2024-11-29 08-59-16](https://github.com/user-attachments/assets/265542a1-c40c-4735-9286-2c82909004a6)

**Comment by 3ab3z2** [comment]
same, I have lots of projects to work on rn and I needed the pack as soon as possible but it's getting late T-T

**Comment by davevad93** [comment]
Hi @tasnim-rizvy ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @tasnim-rizvy was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
